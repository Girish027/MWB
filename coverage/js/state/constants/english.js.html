
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for js/state/constants/english.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../../index.html">All files</a> / <a href="index.html">js/state/constants</a> english.js</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">76.92% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>30/39</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">50% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>1/2</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">55% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>11/20</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">76.92% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>30/39</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line medium'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a>
<a name='L36'></a><a href='#L36'>36</a>
<a name='L37'></a><a href='#L37'>37</a>
<a name='L38'></a><a href='#L38'>38</a>
<a name='L39'></a><a href='#L39'>39</a>
<a name='L40'></a><a href='#L40'>40</a>
<a name='L41'></a><a href='#L41'>41</a>
<a name='L42'></a><a href='#L42'>42</a>
<a name='L43'></a><a href='#L43'>43</a>
<a name='L44'></a><a href='#L44'>44</a>
<a name='L45'></a><a href='#L45'>45</a>
<a name='L46'></a><a href='#L46'>46</a>
<a name='L47'></a><a href='#L47'>47</a>
<a name='L48'></a><a href='#L48'>48</a>
<a name='L49'></a><a href='#L49'>49</a>
<a name='L50'></a><a href='#L50'>50</a>
<a name='L51'></a><a href='#L51'>51</a>
<a name='L52'></a><a href='#L52'>52</a>
<a name='L53'></a><a href='#L53'>53</a>
<a name='L54'></a><a href='#L54'>54</a>
<a name='L55'></a><a href='#L55'>55</a>
<a name='L56'></a><a href='#L56'>56</a>
<a name='L57'></a><a href='#L57'>57</a>
<a name='L58'></a><a href='#L58'>58</a>
<a name='L59'></a><a href='#L59'>59</a>
<a name='L60'></a><a href='#L60'>60</a>
<a name='L61'></a><a href='#L61'>61</a>
<a name='L62'></a><a href='#L62'>62</a>
<a name='L63'></a><a href='#L63'>63</a>
<a name='L64'></a><a href='#L64'>64</a>
<a name='L65'></a><a href='#L65'>65</a>
<a name='L66'></a><a href='#L66'>66</a>
<a name='L67'></a><a href='#L67'>67</a>
<a name='L68'></a><a href='#L68'>68</a>
<a name='L69'></a><a href='#L69'>69</a>
<a name='L70'></a><a href='#L70'>70</a>
<a name='L71'></a><a href='#L71'>71</a>
<a name='L72'></a><a href='#L72'>72</a>
<a name='L73'></a><a href='#L73'>73</a>
<a name='L74'></a><a href='#L74'>74</a>
<a name='L75'></a><a href='#L75'>75</a>
<a name='L76'></a><a href='#L76'>76</a>
<a name='L77'></a><a href='#L77'>77</a>
<a name='L78'></a><a href='#L78'>78</a>
<a name='L79'></a><a href='#L79'>79</a>
<a name='L80'></a><a href='#L80'>80</a>
<a name='L81'></a><a href='#L81'>81</a>
<a name='L82'></a><a href='#L82'>82</a>
<a name='L83'></a><a href='#L83'>83</a>
<a name='L84'></a><a href='#L84'>84</a>
<a name='L85'></a><a href='#L85'>85</a>
<a name='L86'></a><a href='#L86'>86</a>
<a name='L87'></a><a href='#L87'>87</a>
<a name='L88'></a><a href='#L88'>88</a>
<a name='L89'></a><a href='#L89'>89</a>
<a name='L90'></a><a href='#L90'>90</a>
<a name='L91'></a><a href='#L91'>91</a></td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">80x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">// tagging
export const LABEL_TAG_NO_MATCHING = 'No matching tags.';
export const LABEL_TAG_FOUND = 'Matching tags.';
export const LABEL_TAG_NEW = 'This will create a new tag.';
export const LABEL_TAG_ERROR_CREATE = 'To create a new tag, please follow the topic-goal format.';
export const LABEL_TAG_NO_DISPLAY = 'No tags to display.';
export const tagMessages = [
  { status: 1, label: LABEL_TAG_NO_MATCHING }, // 1 is valid topic-goal but not sent
  { status: 2, label: LABEL_TAG_ERROR_CREATE }, // 2 is an error
  { status: 3, label: LABEL_TAG_FOUND }, // 3 is found suggested tags
  { status: 4, LABEL_TAG_NO_DISPLAY },
]; // 4 no suggest tags
export const ERROR_SERVER_500 = 'The server communication error.';
export const ERROR_SERVER_503 = 'Elastic Search is not available.';
export const ERROR_SERVER_400 = 'Bad request.';
export const ERROR_SERVER_401 = 'Unathorized.';
export const ERROR_SERVER_404 = 'Not found.';
export const ERROR_SERVER_405 = 'Not allowed.';
export const ERROR_SERVER_406 = 'Not acceptable.';
export const ERROR_SERVER_409 = 'Already tagged.';
export const ERROR_SERVER_TOO_MANY = 'Too many requests.';
export const ERROR_SERVER_UNKNOWN = 'Server communication error.';
export const ERROR_SERVER_DOWN = 'Your request did not complete in a timely manner. Please try again.';
export const ERROR_TRANSFORM_FAIL = 'Dataset transformation failed.';
&nbsp;
&nbsp;
// Display Messages
&nbsp;
export const DISPLAY_MESSAGES = {
  deleteTransformationItem: itemDeleted =&gt; `Deleted Transformation item: [${JSON.stringify(itemDeleted)}].`,
  tagCreateFail: 'Tag could not be added.',
  tagCreated: 'Tag was successfully added.',
  tagUpdated: 'Tag was successfully updated.',
  tagRemoved: 'Tag was successfully removed.',
  fileImportFail: 'There was a problem importing the file.',
  emptyFile: 'Empty file is uploaded.',
  fileParseFail: 'Error in parsing the data given in the uploaded file, please check documentation for expected format.',
  taggingGuideImportSuccess: validTagCount =&gt; `Successfully imported ${validTagCount} tag${validTagCount == 1 ? <span class="branch-0 cbranch-no" title="branch not covered" >'' </span>: 's'}. Loading now...`,
  configFileUploaded: name =&gt; `Config file '${name}' uploaded.`,
  configAlreadyExists: 'Config already exists.',
  createdSuccess: itemName =&gt; `${itemName} created.`,
  createFailed: 'Create Failed - Are you logged in?',
  projectCreated: projectName =&gt; `Model ${projectName} created.`,
  projectDemoted: 'Model demoted to node level.',
  projectPromoted: 'Model promoted to global level.',
  projectUpdateFailed: 'Model update failed.',
  projectAlreadyExists: 'Model already exists.',
  projectUpdated: 'Model updated.',
  projectDeleted: projectName =&gt; `Model ${projectName} deleted.`,
  projectLatest: <span class="fstat-no" title="function not covered" >cl</span>ientName =&gt; <span class="cstat-no" title="statement not covered" >`Switching to latest project for client ${clientName}.`,</span>
  commentUpdated: 'Comment was successfully updated.',
  commentRemoved: 'Comment was successfully removed.',
  granluarTagRemoved: 'Granular tag removed.',
  bulkTagAdded: <span class="fstat-no" title="function not covered" >in</span>tent =&gt; <span class="cstat-no" title="statement not covered" >`Bulk tag "${intent}" added.`,</span>
  bulkTagUpdated: <span class="fstat-no" title="function not covered" >in</span>tent =&gt; <span class="cstat-no" title="statement not covered" >`Bulk tag "${intent}" updated.`,</span>
  bulkTagAddNUpdate: <span class="fstat-no" title="function not covered" >in</span>tent =&gt; <span class="cstat-no" title="statement not covered" >`Bulk tag "${intent}" added and updated.`,</span>
  bulkTagRemoved: 'Bulk tag removed.',
  invalidIntent: <span class="fstat-no" title="function not covered" >in</span>tent =&gt; <span class="cstat-no" title="statement not covered" >`Tag ${intent} is not in a topic-goal format.`,</span>
  modelAlreadyExists: 'Model version already exists.',
  testResultsRecieved: 'Model test results received.',
  testRequestSubmitted: 'Model batch test request submitted.',
  datasetCreated: <span class="fstat-no" title="function not covered" >da</span>tasetName =&gt; <span class="cstat-no" title="statement not covered" >`Dataset ${datasetName} created.`,</span>
  datasetDeleted: 'Dataset successfully deleted.',
  datasetUsed: 'Cannot delete dataset. It is used in a model.',
  datasetAlreadyExists: <span class="fstat-no" title="function not covered" >da</span>tasetName =&gt; <span class="cstat-no" title="statement not covered" >`Dataset ${datasetName} already exists.`,</span>
  datasetCreateError: 'Dataset creation error.',
  datasetOperationNotAllowed: 'Source type is not allowed for the user.',
  uniqueRollupCount: 'Datasets should have more than 1 unique rollup intents.',
  modelCreated: itemName =&gt; `Model version ${itemName} created.`,
  modelUpdated: itemName =&gt; `Model version ${itemName} updated.`,
  modelDeleted: itemName =&gt; `Model version ${itemName} deleted.`,
  markModelFlagSuccess: <span class="fstat-no" title="function not covered" >mo</span>delVersion =&gt; <span class="cstat-no" title="statement not covered" >`Model version ${modelVersion} is marked for deploy.`,</span>
  markModelFlagFailed: modelVersion =&gt; `Unable to mark model version ${modelVersion} as deployable.`,
  unmarkModelFlagSuccess: <span class="fstat-no" title="function not covered" >mo</span>delVersion =&gt; <span class="cstat-no" title="statement not covered" >`Model version ${modelVersion} is unmarked for deploy.`,</span>
  unmarkModelFlagFailed: modelVersion =&gt; `Unable to unmark model version ${modelVersion} from deployable.`,
  modelAlreadyDeleted: 'Model is already deleted.',
  modelDeleteFailed: 'Model deletion failed.',
  buildModelRequest: 'Build request is submitted.',
  audioFileNotAvailable: 'Something went wrong. Please retry again.',
  speechTestTimeoutMsg: 'Your request timed out. Please retry again',
  modelTestConnectionError: 'Could not process your request at this moment. Please check with the admin.',
  modelTestFailure: 'Could not process your request. Please retry again.',
  audioRecordFail: 'Unable to record audio at this time.',
  micPermissionDenied: 'Permission denied to record audio.',
  fileDownloadFailed: 'Unable to download file.',
  audioNotPlayable: 'Unable to play audio file.',
  evalSpeechFailed: 'Failed to evaluate the utterance.',
  invalidConfigFile: 'Invalid File. Please upload a valid json file.',
  sessionExpired: 'Your session is expired. Please login again.',
};
&nbsp;</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Thu Aug 05 2021 15:06:58 GMT+0530 (IST)
            </div>
        </div>
        <script src="../../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../../sorter.js"></script>
        <script src="../../../block-navigation.js"></script>
    </body>
</html>
    