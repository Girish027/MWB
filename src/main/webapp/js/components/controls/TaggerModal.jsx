import React, { Component } from 'react';
import PropTypes from 'prop-types';
import Box from 'grommet/components/Box';
import Layer from 'grommet/components/Layer';


export default class TaggerModal extends Component {
  constructor(props) {
    super(props);
    this.props = props;
  }

  render() {
    const {
      id, resize, header, content, actions, className,
    } = this.props;
    return (
      <Layer
        align="center"
        closer={false}
        id="TaggerModal"
        className={
          `TaggerControl TaggerModal${
            resize ? ' TaggerModalResize' : ''
          }${className ? ` ${className}` : ''}`
        }
      >
        <Box direction="column" id={id} flex>
          <Box className="ModalRow ModalHead">
            <span className="ModalTitle">
              {typeof header === 'function' ? header() : header}
            </span>
            {/* <span className="ModalClose"> X </span> */}
          </Box>
          <Box className="ModalRow ModalContent" flex>
            {typeof content === 'function' ? content() : content}
          </Box>
          <Box className="ModalRow ModalActions" direction="row" justify="end">
            {typeof actions === 'function' ? actions() : actions}
          </Box>
        </Box>
      </Layer>
    );
  }
}

TaggerModal.propTypes = {
  id: PropTypes.string,
  resize: PropTypes.bool,
  header: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
  content: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
  actions: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
  className: PropTypes.string,
};
