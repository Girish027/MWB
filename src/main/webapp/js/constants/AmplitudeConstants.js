
const amplitudeConstants = {

  // ToolId for Modeling Work Bench
  TOOL_ID: 'MWB',

  // List of Events
  LOGIN_EVENT: 'LoginUser',
  LOGOUT_EVENT: 'LogoutUser',
  SELECT_CLIENT_EVENT: 'SelectClientApp',
  CREATE_DATASET_EVENT: 'CreateDataset',
  CREATE_DATASET_COMPLETED_EVENT: 'CreateDatasetCompleted',
  CREATE_DATASET_FAILED_EVENT: 'CreateDatasetFailed',
  DELETE_DATASET_EVENT: 'DeleteDataset',
  DELETE_DATASET_COMPLETED_EVENT: 'DeleteDatasetCompleted',
  DELETE_DATASET_FAILED_EVENT: 'DeleteDatasetFailed',
  CREATE_MODEL_EVENT: 'CreateModel',
  CREATE_MODEL_COMPLETED_EVENT: 'CreateModelCompleted',
  CREATE_MODEL_FAILED_EVENT: 'CreateModelFailed',
  DELETE_MODEL_EVENT: 'DeleteModel',
  DELETE_MODEL_COMPLETED_EVENT: 'DeleteModelCompleted',
  DELETE_MODEL_FAILED_EVENT: 'DeleteModelFailed',
  TEST_SPEECH_MODEL_EVENT: 'TestSpeechModel',
  TEST_SPEECH_MODEL_COMPLETED_EVENT: 'TestSpeechModelCompleted',
  TEST_SPEECH_MODEL_FAILED_EVENT: 'TestSpeechModelFailed',
  TEST_DIGITAL_MODEL_EVENT: 'TestDigitalModel',
  TEST_DIGITAL_MODEL_COMPLETED_EVENT: 'TestDigitalModelCompleted',
  DIGITAL_MODEL_TEST_FAILED_EVENT: 'TestDigitalModelFailed',
  DOWNLOAD_MODEL_EVENT: 'DownloadModel',
  IMPORT_TAGGING_GUIDE_EVENT: 'ImportTaggingGuide',
  IMPORT_TAGGING_GUIDE_COMPLETED_EVENT: 'ImportTaggingGuideCompleted',
  IMPORT_TAGGING_GUIDE_FAILED_EVENT: 'ImportTaggingGuideFailed',
  RUN_BATCH_TEST_EVENT: 'RunBatchTest',
};

export default amplitudeConstants;
