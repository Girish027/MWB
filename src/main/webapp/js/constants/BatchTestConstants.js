
const PREDICTED_RUI = 'Predicted RUI';
const batchTestConstants = {
  DEFAULT_PAGE_SIZE: 100,

  // Filter Related
  TRANSCRIPTIONS_FILTER_NAME: 'Transcriptions',
  TRANSCRIPTIONS_FILTER_LIST: ['All', 'Misclassified'],
  INTENT_FILTER_NAME: 'Intent',
  FILTER_SELECTED_ALL: 'All',
  ROLLUP_INTENT_ROW_ID: 'Rollup Intent (RUI)',
  PREDICTED_RUI_1: `${PREDICTED_RUI} 1`,
  PREDICTED_RUI_2: `${PREDICTED_RUI} 2`,
  PREDICTED_RUI_3: `${PREDICTED_RUI} 3`,
  PREDICTED_RUI_4: `${PREDICTED_RUI} 4`,
  PREDICTED_RUI_5: `${PREDICTED_RUI} 5`,
};

export default batchTestConstants;
