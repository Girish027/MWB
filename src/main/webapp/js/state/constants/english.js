// tagging
export const LABEL_TAG_NO_MATCHING = 'No matching tags.';
export const LABEL_TAG_FOUND = 'Matching tags.';
export const LABEL_TAG_NEW = 'This will create a new tag.';
export const LABEL_TAG_ERROR_CREATE = 'To create a new tag, please follow the topic-goal format.';
export const LABEL_TAG_NO_DISPLAY = 'No tags to display.';
export const tagMessages = [
  { status: 1, label: LABEL_TAG_NO_MATCHING }, // 1 is valid topic-goal but not sent
  { status: 2, label: LABEL_TAG_ERROR_CREATE }, // 2 is an error
  { status: 3, label: LABEL_TAG_FOUND }, // 3 is found suggested tags
  { status: 4, LABEL_TAG_NO_DISPLAY },
]; // 4 no suggest tags
export const ERROR_SERVER_500 = 'The server communication error.';
export const ERROR_SERVER_503 = 'Elastic Search is not available.';
export const ERROR_SERVER_400 = 'Bad request.';
export const ERROR_SERVER_401 = 'Unathorized.';
export const ERROR_SERVER_404 = 'Not found.';
export const ERROR_SERVER_405 = 'Not allowed.';
export const ERROR_SERVER_406 = 'Not acceptable.';
export const ERROR_SERVER_409 = 'Already tagged.';
export const ERROR_SERVER_TOO_MANY = 'Too many requests.';
export const ERROR_SERVER_UNKNOWN = 'Server communication error.';
export const ERROR_SERVER_DOWN = 'Your request did not complete in a timely manner. Please try again.';
export const ERROR_TRANSFORM_FAIL = 'Dataset transformation failed.';


// Display Messages

export const DISPLAY_MESSAGES = {
  deleteTransformationItem: itemDeleted => `Deleted Transformation item: [${JSON.stringify(itemDeleted)}].`,
  tagCreateFail: 'Tag could not be added.',
  tagCreated: 'Tag was successfully added.',
  tagUpdated: 'Tag was successfully updated.',
  tagRemoved: 'Tag was successfully removed.',
  fileImportFail: 'There was a problem importing the file.',
  emptyFile: 'Empty file is uploaded.',
  fileParseFail: 'Error in parsing the data given in the uploaded file, please check documentation for expected format.',
  taggingGuideImportSuccess: validTagCount => `Successfully imported ${validTagCount} tag${validTagCount == 1 ? '' : 's'}. Loading now...`,
  configFileUploaded: name => `Config file '${name}' uploaded.`,
  configAlreadyExists: 'Config already exists.',
  createdSuccess: itemName => `${itemName} created.`,
  createFailed: 'Create Failed - Are you logged in?',
  projectCreated: projectName => `Model ${projectName} created.`,
  projectDemoted: 'Model demoted to node level.',
  projectPromoted: 'Model promoted to global level.',
  projectUpdateFailed: 'Model update failed.',
  projectAlreadyExists: 'Model already exists.',
  projectUpdated: 'Model updated.',
  projectDeleted: projectName => `Model ${projectName} deleted.`,
  projectLatest: clientName => `Switching to latest project for client ${clientName}.`,
  commentUpdated: 'Comment was successfully updated.',
  commentRemoved: 'Comment was successfully removed.',
  granluarTagRemoved: 'Granular tag removed.',
  bulkTagAdded: intent => `Bulk tag "${intent}" added.`,
  bulkTagUpdated: intent => `Bulk tag "${intent}" updated.`,
  bulkTagAddNUpdate: intent => `Bulk tag "${intent}" added and updated.`,
  bulkTagRemoved: 'Bulk tag removed.',
  invalidIntent: intent => `Tag ${intent} is not in a topic-goal format.`,
  modelAlreadyExists: 'Model version already exists.',
  testResultsRecieved: 'Model test results received.',
  testRequestSubmitted: 'Model batch test request submitted.',
  datasetCreated: datasetName => `Dataset ${datasetName} created.`,
  datasetDeleted: 'Dataset successfully deleted.',
  datasetUsed: 'Cannot delete dataset. It is used in a model.',
  datasetAlreadyExists: datasetName => `Dataset ${datasetName} already exists.`,
  datasetCreateError: 'Dataset creation error.',
  datasetOperationNotAllowed: 'Source type is not allowed for the user.',
  uniqueRollupCount: 'Datasets should have more than 1 unique rollup intents.',
  modelCreated: itemName => `Model version ${itemName} created.`,
  modelUpdated: itemName => `Model version ${itemName} updated.`,
  modelDeleted: itemName => `Model version ${itemName} deleted.`,
  markModelFlagSuccess: modelVersion => `Model version ${modelVersion} is marked for deploy.`,
  markModelFlagFailed: modelVersion => `Unable to mark model version ${modelVersion} as deployable.`,
  unmarkModelFlagSuccess: modelVersion => `Model version ${modelVersion} is unmarked for deploy.`,
  unmarkModelFlagFailed: modelVersion => `Unable to unmark model version ${modelVersion} from deployable.`,
  modelAlreadyDeleted: 'Model is already deleted.',
  modelDeleteFailed: 'Model deletion failed.',
  buildModelRequest: 'Build request is submitted.',
  audioFileNotAvailable: 'Something went wrong. Please retry again.',
  speechTestTimeoutMsg: 'Your request timed out. Please retry again',
  modelTestConnectionError: 'Could not process your request at this moment. Please check with the admin.',
  modelTestFailure: 'Could not process your request. Please retry again.',
  audioRecordFail: 'Unable to record audio at this time.',
  micPermissionDenied: 'Permission denied to record audio.',
  fileDownloadFailed: 'Unable to download file.',
  audioNotPlayable: 'Unable to play audio file.',
  evalSpeechFailed: 'Failed to evaluate the utterance.',
  invalidConfigFile: 'Invalid File. Please upload a valid json file.',
  sessionExpired: 'Your session is expired. Please login again.',
};
