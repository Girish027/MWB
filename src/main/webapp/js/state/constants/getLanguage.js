import config from 'config';
import * as english from './english';

export const getLanguage = () => {
  switch (config.lang) {
  case 'english':
  default:
    return english;
  }
};
