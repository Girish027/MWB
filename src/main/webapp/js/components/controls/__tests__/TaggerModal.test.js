import React from 'react';
import { shallow, mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import { getLanguage } from 'state/constants/getLanguage';
import TaggerModal from 'components/controls/TaggerModal';


describe('<TaggerModal />', () => {
  let wrapper;

  const props = {
    header: 'Apply Suggested Intents',
  };

  describe('Creating an instance with no props', () => {
    beforeEach(() => {
      wrapper = shallow(<TaggerModal />);
    });
    test('should exist', () => {
      expect(wrapper.exists()).toBe(true);
    });

    test('Snapshots should render correctly', () => {
      expect(toJSON(wrapper)).toMatchSnapshot();
    });
  });

  describe('Creating an instance with test props', () => {
    beforeEach(() => {
      wrapper = mount(<TaggerModal
        {...props}
      />);
    });

    test('should exist', () => {
      expect(wrapper.exists()).toBe(true);
    });

    test('Snapshots should render correctly', () => {
      expect(toJSON(wrapper)).toMatchSnapshot();
    });
  });
});
