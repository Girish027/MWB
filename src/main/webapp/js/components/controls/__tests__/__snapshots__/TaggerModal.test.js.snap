// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TaggerModal /> Creating an instance with no props Snapshots should render correctly 1`] = `
<Layer
  align="center"
  className="TaggerControl TaggerModal"
  closer={false}
  id="TaggerModal"
>
  <Box
    announce={false}
    direction="column"
    flex={true}
    focusable={true}
    pad="none"
    responsive={true}
    tag="div"
  >
    <Box
      announce={false}
      className="ModalRow ModalHead"
      direction="column"
      focusable={true}
      pad="none"
      responsive={true}
      tag="div"
    >
      <span
        className="ModalTitle"
      />
    </Box>
    <Box
      announce={false}
      className="ModalRow ModalContent"
      direction="column"
      flex={true}
      focusable={true}
      pad="none"
      responsive={true}
      tag="div"
    />
    <Box
      announce={false}
      className="ModalRow ModalActions"
      direction="row"
      focusable={true}
      justify="end"
      pad="none"
      responsive={true}
      tag="div"
    />
  </Box>
</Layer>
`;

exports[`<TaggerModal /> Creating an instance with test props Snapshots should render correctly 1`] = `
<TaggerModal
  header="Apply Suggested Intents"
>
  <Layer
    align="center"
    className="TaggerControl TaggerModal"
    closer={false}
    id="TaggerModal"
  >
    <span
      style={
        Object {
          "display": "none",
        }
      }
    />
  </Layer>
</TaggerModal>
`;
